src/app/guards/no-auth.guard.spec.ts

```diff
@@ -1,40 +1,45 @@
+import { localStorageSpy } from './../helpers/spies';
```

Comment:
> Since we've been using absolute imports for our files, you can also do the same here for consistency.

Same for here.

Reply:
Thanks, @wangonya , I have worked on this